modules/ibmz-configure-nbde-with-static-ip.adoc

```diff
@@ -120,35 +120,36 @@ ifndef::ibm-z-kvm[]
 ----
 rd.neednet=1 \
 console=ttysclp0 \
-coreos.inst.install_dev=/dev/dasda \ <1>
+coreos.inst.install_dev=/dev/dasda \// <1>
 ignition.firstboot ignition.platform.id=metal \
-coreos.live.rootfs_url=http://10.19.17.25/redhat/ocp/rhcos-413.86.202302201445-0/rhcos-413.86.202302201445-0-live-rootfs.s390x.img \
-coreos.inst.ignition_url=http://bastion.ocp-cluster1.example.com:8080/ignition/master.ign \
+coreos.live.rootfs_url=<rhcos_liveos> \// <2>
```

Comment:
I do not like this name. 
1. I would prefer "rhcos_rootfs" as short name for "rhcos live rootfs" image name
2. The value requires an url (with rhcos live rootfs file name).
```suggestion
coreos.live.rootfs_url=<rhcos_rootfs_url> \// <2>
```
Therefor, I would prefer to use "rhcos_rootfs_url" as variable, instead of "rhcos_liveos"

Reply:
changing as discussed in the huddle